### PR TITLE
Web-based image panel

### PR DESCRIPTION
**Pull request by Atrejoe**
A generic web-based image panel, that allows argument substitution

### CODE REVIEW

**Comment by aceisace** [comment]
Some conditional checks would be great (for example if the URL doesn't point to a (supported) image). Can I propose some minor changes?

File: modules/inkycal_webimage.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+"""
+Third-party (custom) module for Inky-Calendar software.
+What it does: obtains an image from a URL. As restriction to this image apply, only use images that have been generated for inkycal.
+Copyright by Robert Sirre
+"""
+from __future__ import print_function # Required
+from configuration import * # Required
+import requests
+
+# Any static variables go here. For example, the language.
```
Comment (vitasam) [inline modules/inkycal_webimage.py]: It will be great to remove comments which are remaining from template module

**Comment by Atrejoe** [comment]
> Some conditional checks would be great (for example if the URL doesn't point to a (supported) image). Can I propose some minor changes?

@aceisace yes please

**Comment by aceisace** [comment]
Closing this PR. The new inkycal_image module substitutes this one as it contains a few more checks, automatic orientation and a few more configuration options. Nonetheless, thank you @Atrejoe for taking the time to develop this panel :)

**Pull request closed without merging**